test/functional/services/remote/webdriver.ts

```diff
@@ -39,6 +39,7 @@ const headlessBrowser: string = process.env.TEST_BROWSER_HEADLESS as string;
 const browserBinaryPath: string = process.env.TEST_BROWSER_BINARY_PATH as string;
 const remoteDebug: string = process.env.TEST_REMOTE_DEBUG as string;
 const certValidation: string = process.env.NODE_TLS_REJECT_UNAUTHORIZED as string;
+const noCache: string = process.env.DISABLE_CACHE as string;
```

Comment:
Similar as [here](https://github.com/elastic/kibana/pull/119681/files#r782631721). Mind if we go with something like `TEST_DISABLE_CACHE` to match the same pattern?

Reply:
Agreed 👍🏼